addons/cookoff/initSettings.sqf

```diff
@@ -10,6 +10,16 @@
     true // Needs mission restart
 ] call CBA_settings_fnc_init;
 
+[
+    QGVAR(disableForAI), "CHECKBOX",
+    [LSTRING(disableForAI_name), LSTRING(disableForAI_tooltip)],
+    LSTRING(category_displayName),
+    false, // default value
+    true, // isGlobal
+    {[QGVAR(disableForAI), _this] call EFUNC(common,cbaSettings_settingChanged)},
+    true // Needs mission restart
+] call CBA_settings_fnc_init;
```

Comment:
Okay, so why is this a "disable" type setting, when the others are "enable"?
And it is by default off, so the default is a double negative "don't deactive". Would it not make more sense to have it say "activate" and just invert the default value?

Also, there already is a enable setting. I'd rather change the enable setting from checkbox to list with the following options:
- enable
- disable
- enable for player controlled vehicles

because those are the only 3 logical distinct options (`disable, enableAI` and `disable, disableAI` have the same effect).

List type settings should be bw-compatible with setting values like false and true with
flase -> index 0
true -> index 1

if not I'd consider it a bug in cba_settings.

Reply:
Also CBA_fnc_addSetting.